index.html

```diff
@@ -7,10 +7,85 @@
 
   <link href="https://fonts.googleapis.com/css?family=Bangers|Titillium+Web" rel="stylesheet">
   <link rel="stylesheet" href="css/index.css">
-
+  <meta name="viewport" content="width=device-width, initial-scale=1">
 </head>
 
 <body>
 <!-- Use past code you created here for Great Idea Web Page or past solution for Great Idea Web Page HTML here! -->
+<header class= "header">
+  <div class="head">
+    <nav>
+      <a href="#">Services</a>
+      <a href="#">Product</a>
+      <a href="#">Vision</a>
+      <a href="#">Features</a>
+      <a href="#">About</a>
+      <a href="#">Contact</a>
+    </nav>
+  </div>
+
+  <div class="head">
+    <img class="logo" src="https://github.com/LambdaSchool/User-Interface/blob/master/great-idea-website/img/logo.png?raw=true" alt="Great Idea! Company logo.">
+  </div>
+</header>
+<div class="title-pic">
+  <div class="top">
+    <h1>Innovation<br> On<br> Demand</h1>
+    <button>Get Started</button>
+  </div>
+  <img src="https://github.com/LambdaSchool/User-Interface/blob/master/great-idea-website/img/header-img.png?raw=true" alt="Image of a code snippet.">
+</div>
+<hr>
+<div class="feature-about">
+  <div class="top-mid">
+    <h2>Features </h2>
+    <p>Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>
+  </div>
+
+  <div class="top-mid">
+    <h2>About</h2>
+    <p>Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>
+  </div>
+</div>
+
+<div class="center-pic">
+  <img class="middle-img" src="https://github.com/LambdaSchool/User-Interface/blob/master/great-idea-website/img/mid-page-accent.jpg?raw=true" alt="Image of code snippets across the screen">
+</div>
+<div class="bottom-text">
+  <div class="bottom-mid">
+    <h2>Services</h2>
+    <p>Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis. </p>
+  </div>
+
+  <div class="bottom-mid">
+    <h2>Product</h2>
+    <p>Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>
+  </div>
+
+  <div class="bottom-mid">
+    <h2>Vision</h2>
+    <p>Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>
+  </div>
+</div>
+
+
+  <hr>
+
+  <div class="contact">
+  <h2>Contact</h2>
+  <address>
+  123 Way 456 Street<br>
+  Somewhere, USA 1
+  <br>
+  <br>
+  (888) 888-8888
+  <br>
+  <br>
```

Comment:
Using `br` tags for spacing is a quick and dirty way to make space between elements. 

Since this is styling along with the `hr` tag above should all be done in the CSS. 

Try applying a `span` or `strong` tag around what you are trying to style and adjust the `line-height`, or `margins`, or `padding`.